src/modules/taxonomy/taxonomyApi.ts

```diff
@@ -22,48 +19,81 @@ import {
   ResourceType,
   TaxonomyMetadata,
 } from './taxonomyApiInterfaces';
+import { WithTaxonomyVersion } from '../../interfaces';
 
 const baseUrl = apiResourceUrl(taxonomyApi);
 
+const { fetchAndResolve } = httpFunctions;
+
+const stringifyQuery = (object: Record<string, any> = {}) => {
+  const stringified = `?${queryString.stringify(object)}`;
+  return stringified === '?' ? '' : stringified;
```

Comment:
Ser du har denne fleire plasser. Er ? i første linje for å passe på at det ikkje er undefined og dermed for å kunne sjekke på '?' i andre linje?

Reply:
`query-string` fjerner alle `undefined`-verdier, så dersom du passerer inn `{language: undefined}` vil output fra linje 1 være `?`. Da er det enklere å bare returnere en tom streng, slik at man ikke ender opp med f.eks `/resource-types?`

Reply:
Det skjønner eg. Det var meir måten det var gjort på som eg stussa på.  Eg hadde sikkert gjort nokke sånt, uten at det er bedre, berre litt anderledes.
```
const stringified = queryString.stringify(object);
return stringified ?  `?${stringified}` : '';
```